src/test/java/com/google/api/codegen/testdata/fakebilling_gapic.yaml

```diff
@@ -69,8 +69,8 @@ interfaces:
   lang_doc:
     java: Configure billing projects.
   collections:
-  - name_pattern: billingAccounts/{billingAccountsId}
-    entity_name: billingAccountsId
+  - name_pattern: v1/projects/{projectsId}/billingInfo
+    entity_name: projectsId
```

Comment:
Changed yaml entity names to be lower underscore singular and changed entity patterns to be [resources]/[resource], without version number etc.